site/source/tutorials/1010_containers_and_injecting_state/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: tutorial
+title: State management
+overview: In this tutorial, you will learn what Containers and Injectors are and how to use them to coordinate external state and inject as properties into the sections of the widget tree.
+---
+
+{% section 'first' %}
+
+# State management
+
+## Overview
+
+Modern web application today are required to manage complex state models which can involve fetching data from a remote service or multiple widget requiring the same slices of state. While Dojo 2's widgets can manage application state, encapsulation and a clean separation of concerns may be lost if widgets manage their own visual representations, listen for interactions from the user, manage their children, and keep track of state information. Additionally, using widgets to pass state through an application often forces the widgets to be aware of state information for the sole purpose of passing that data down to their children. To allow widgets to remain focused on their primary roles of providing a visual representation of themselves and listening for user interactions, Dojo 2 provides two classes, `Container`, and `Injector` that are designed to coordinate an application's external state and connect and map this state to properties only for the widgets that it is required.
```

Comment:
"or multiple widget requiring the same" -> "or multiple widgets requiring the same"

Reply:
" Dojo 2 provides two classes, `Container`, and `Injector` that" ->  "Dojo 2 provides two classes, `Container` and `Injector`, that"

Reply:
"widgets that it is required" -> something is odd with this tense, maybe reword it to be more explicit?